ddpui/utils/helpers.py

```
@@ -220,3 +221,66 @@ def get_integer_env_var(varname: str, default_value: int, logger, allow_negative
+        prerelease = match.group(4)
+        return (major, minor, patch, prerelease)
+
+    def cmp(a, b):
+        """Returns 1, 0, or -1"""
+        return (a > b) - (a < b)
+
+    v1 = parse_version(version1)
+    v2 = parse_version(version2)
+
+    # Compare major, minor, patch
+    for a, b in zip(v1[:3], v2[:3]):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)